firmware/shared/src/hw/hw_can_h7.c

```diff
@@ -24,8 +28,9 @@ void hw_can_init(CanHandle *can_handle)
     // Configure interrupt mode for CAN peripheral.
     assert(
         HAL_FDCAN_ActivateNotification(
-            can_handle->hcan, FDCAN_IT_RX_FIFO0_NEW_MESSAGE | FDCAN_IT_RX_FIFO1_NEW_MESSAGE, 0) == HAL_OK);
-    assert(HAL_FDCAN_ActivateNotification(can_handle->hcan, FDCAN_IT_BUS_OFF, 0) == HAL_OK);
+            can_handle->hcan,
+            FDCAN_IT_RX_FIFO0_NEW_MESSAGE | FDCAN_IT_RX_FIFO1_NEW_MESSAGE | FDCAN_IT_BUS_OFF | FDCAN_IT_TX_COMPLETE,
+            1) == HAL_OK);
```

Comment:
note that this only activates the notifcatoin on bufferindex = 1. I think there's only one tx queue so it shoudln't be a problem but...